galsim/fitswcs.py

```diff
@@ -1451,15 +1422,14 @@ def _local(self, image_pos, color=None):
             order = len(self.ab[0])-1
             xpow = x ** np.arange(order+1)
             ypow = y ** np.arange(order+1)
-            p1 += np.dot(np.dot(self.ab, ypow), xpow)
+            p1 = np.dot(np.dot(self.ab, ypow), xpow)
```

Comment:
I thought so.  I tried it, but it made it slower.  Probably because x,y are always scalars here, so the overhead of turning things into numpy arrays of length 1 in the horner2d code ended up being more than making the short xpow and ypow arrays.